experiments/process_sandbox/host_service_calls.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+/**
+ * This file includes all of the types used for forwarding memory-provider
+ * requests and pagemap updates from the sandbox to the parent.  This is a
+ * trivial lightweight request-response protocol, using fixed-size binary
+ * messages.
+ *
+ * This protocol is trivial and so is done as something very simple and ad-hoc.
+ * It should be replaced by something more robust if it ever grows more
+ * complex. After sandboxes are made reentrant, the same upcall mechanism used
+ * for calling back into Verona can be used to replace this.
+ */
+
+namespace sandbox
+{
+  /**
+   * The ID of the method that's being proxied.
+   */
+  enum HostServiceCallID : uintptr_t
+  {
+    /**
+     * Push a large allocation to the stack.  The first argument is the address
+     * of the slab, the second the large sizeclass.  The return value is unused.
```

Comment:
the address of the slab was returned by reserve below, right? Perhaps add that to the comment.